services/storage/src/simcore_service_storage/modules/celery/client.py

```diff
@@ -0,0 +1,117 @@
+import logging
+from typing import Any, Final
+from uuid import uuid4
+
+from celery import Celery
+from celery.contrib.abortable import AbortableAsyncResult
+from common_library.async_tools import make_async
+from models_library.progress_bar import ProgressReport
+from pydantic import ValidationError
+
+from .models import TaskContext, TaskID, TaskStatus, TaskUUID
+
+_logger = logging.getLogger(__name__)
+
+_CELERY_INSPECT_TASK_STATUSES = (
+    "active",
+    "registered",
+    "scheduled",
+    "revoked",
+)
+_CELERY_TASK_META_PREFIX = "celery-task-meta-"
+_CELERY_TASK_ID_PREFIX: Final[str] = "ct"  # short for celery task, not Catania
```

Comment:
any reason to not use `celery-task-` as a prefix instead of `ct`? In the context of Z43 `CT` usually means computer tomography. Short one-letter names are cool in programming languages like fortran or k but I think in python long names are reasonable. If you are on the other hand bound by length limitations (like, for example, is the celery task id ever used as a fully qualified dns name?), then of course pick something short. But maybe even `celery-` is a bit more clear than `ct-` ;)

Reply:
Also, this might be a premature overoptimization (so consider it `minor`): Should this maybe be configurable via env-vars instead of being a hardcoded char?